grpc.go

```diff
@@ -310,6 +356,24 @@ func (a *agentGRPC) runProcess(cid string, agentProcess *pb.Process) (pid int, e
 	return pid, nil
 }
 
+func (a *agentGRPC) updateContainerConfig(spec *specs.Spec, config *configs.Config) error {
+	// Update shared PID namespace.
+	for idx, ns := range config.Namespaces {
+		if ns.Type == configs.NEWPID {
+			// In case the path is empty because we don't expect
+			// the containers to share the same PID namespace, a
+			// new PID ns is going to be created.
+			config.Namespaces[idx].Path = a.sandbox.sharedPidNs.path
+			break
+		}
+	}
+
+	// Add the value for NoNewPrivileges option.
+	config.NoNewPrivileges = spec.Process.NoNewPrivileges
+
```

Comment:
I'd prefere to see 2 different routines, one for updating the PID ns path, and another one for the privileges.

Reply:
Done